istioctl/pkg/writer/pilot/testdata/multiStatusMultiPilot.txt

```diff
@@ -1,3 +1,3 @@
-PROXY		STATUS		SENT				ACKNOWLEDGED			PILOT
-proxy1		DIVERGED	2009-11-10 23:00:00 UTC		2009-11-10 22:00:00 UTC		pilot2
-proxy2		SYNCED		2009-11-10 23:00:00 UTC		2009-11-10 23:00:00 UTC		pilot1
\ No newline at end of file
+PROXY		CDS		LDS		EDS			RDS		PILOT
+proxy1		DIVERGED	SYNCED		SYNCED (100%)		NOT SENT	pilot1
```

Comment:
what does diverged mean here? Either proxy is lagging behind or upto date. May be STALE is a better term here?

Reply:
Agreed, changed.